superset/config.py

```diff
@@ -630,7 +630,7 @@ def _try_json_readsha(  # pylint: disable=unused-argument
 
 # Default row limit for SQL Lab queries. Is overridden by setting a new limit in
 # the SQL Lab UI
-DEFAULT_SQLLAB_LIMIT = 1000
+DEFAULT_SQLLAB_LIMIT = 10000
```

Comment:
@betodealmeida any reason for the change from 1,000 to 10,000? We have had a number of employees mention that 10,000 overloads their local storage use and breaks Chrome.

Reply:
I'm not sure, let me change the default back. Sorry for that!